src/gdbstub_impl/ext/reverse_exec.rs

```diff
@@ -0,0 +1,106 @@
+use super::prelude::*;
+use crate::protocol::{
+    commands::ext::{ReverseCont, ReverseStep},
+    SpecificIdKind,
+};
+
+use crate::target::ext::base::BaseOps;
```

Comment:
nit: For consistency, can you match the import order +style used in other modules?

e.g: https://github.com/daniel5151/gdbstub/blob/3084b31/src/gdbstub_impl/ext/base.rs#L1-L8

- group 1 should have the prelude + the specific ext(s) this file implements handler(s) for
- group 2 has the rest of the crate-level imports